src/main/java/controller/lotto/LottoController.java

```diff
@@ -3,40 +3,59 @@
 import controller.lotto.view.LottoInputView;
 import controller.lotto.view.LottoResultView;
 import domain.common.Money;
+import domain.lotto.LottoCount;
 import domain.lotto.LottoFactory;
 import domain.lotto.UserLotto;
 import domain.lotto.WinningLotto;
+import domain.lotto.genrate.CustomLottoGenerator;
+import domain.lotto.genrate.LottoGenerator;
 import domain.lotto.genrate.RandomGenerator;
 
 import java.util.List;
 
+import static domain.lotto.properties.LottoProperties.LOTTO_PRICE;
+
 public class LottoController {
-    /**
-     * 로또를 랜덤으로 구매한다.
-     *
-     * @param args
-     */
+    private static final Money lottoMoney = new Money(LOTTO_PRICE);
+
+
     public static void main(String[] args) {
         Money money = Money.from(LottoInputView.createMoneyView());
 
-        UserLotto userLotto = createUserLotto(money);
+        LottoCount lottoCount = createCustomLottoCount(money);
+
+        CustomLottoGenerator customLottoGenerator = createCustomLottoGenerator(lottoCount.customLottoCount());
+
+        UserLotto userLotto = createUserLotto(List.of(customLottoGenerator, new RandomGenerator()), lottoCount);
 
-        LottoResultView.createUserLottoListView(userLotto.lottoList());
+        LottoResultView.createUserLottoListView(userLotto.lottoList(), lottoCount);
 
         WinningLotto winningLotto = createWinningLotto();
 
         LottoResultView.createStatisticsView(LottoStatistics.from(userLotto, winningLotto), money);
 
     }
 
+    private static CustomLottoGenerator createCustomLottoGenerator(Integer customLottoCount) {
+        return CustomLottoGenerator.from(LottoInputView.createCustomLottoIntegerList(customLottoCount));
+    }
+
+    private static LottoCount createCustomLottoCount(Money money) {
```

Comment:
createCustomLottoCount인데 autoLottoCount도 포함하고 있어서 다소 어색한 것 같습니다.

```suggestion
    private static LottoCount createLottoCount(Money money) {
```